invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/compileCatalog.js

```diff
@@ -0,0 +1,39 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
```

Comment:
```suggestion
// This file is part of invenio-Requests
// Copyright (C) 2022 Graz University of Technology.
// Copyright (C) 2022 CERN
```
If we have this same code copy/pasted across different repositories maybe we could consider having it in an i18n repo, I see that we have [invenio-i18n](https://github.com/inveniosoftware/invenio-i18n), that could be a good candidate, WDYT?

Reply:
lets discuss on the standup, but I think it is a good idea